applications/asset_tracker/src/main.c

```diff
@@ -2095,13 +2165,61 @@ static void lte_handler(const struct lte_lc_evt *const evt)
 	case LTE_LC_EVT_CELL_UPDATE:
 		LOG_INF("LTE cell changed: Cell ID: %d, Tracking area: %d",
 			evt->cell.id, evt->cell.tac);
-#if defined(CONFIG_NRF_CLOUD_CELL_POS)
+#if defined(CONFIG_NRF_CLOUD_CELL_POS) && !defined(CONFIG_CELL_POS_MULTICELL)
 		/* Request location based on new network info */
 		if (data_send_enabled() && evt->cell.id != -1) {
 			k_work_reschedule_for_queue(&application_work_q,
 						       &send_cell_pos_request_work,
 						       K_SECONDS(1));
 		}
+#endif
+		break;
+	case LTE_LC_EVT_NEIGHBOR_CELL_MEAS:
+		LOG_INF("Neighbor cell measurements received: %d",
+			evt->cells_info.ncells_count);
+#if defined(CONFIG_CELL_POS_MULTICELL)
+		if (!data_send_enabled()) {
+			LOG_DBG("Cannot send data; ignoring measurements.");
+			break;
+		}
+		if (evt->cells_info.current_cell.id == 0) {
```

Comment:
This check will need an update if #5655 goes in first

Reply:
☝️ PR has been merged

Reply:
yes, thanks, about to rebase...